.github/workflows/build.yml

```diff
@@ -1,15 +1,11 @@
-name: Builds the website (doesn't deploy - only used for checking syntax)
+name: Build the website (doesn't deploy - only used for checking syntax)
 
 on:
   push
```

Comment:
I just realised I messed this up when I first added the action, in this PR could you change:
```yaml
on:
    push
```
to
```yaml
on: [push, pull_request]
```

in this PR so we don't have to fix it later

Reply:
I just want to see the CI running then I can merge this, it turns out `on: push` doesn't trigger for pull requests from another repo

Reply:
Done.

I'll have to refactor this PR after merging the YAML Lint one, as there will be merge conflicts due to indentation.